### ISSUE TITLE
SliverGrid  inside  CustomScrollView doesn't work after updating from 1.3.8 to 1.3.9

### ISSUE THREAD

**Issue by nam-dh** [peng8350/flutter_pulltorefresh#96]
Your library is great. I really like it.

I'm using `SliverGrid` for displaying grid of items and followed your guideline to add it inside a `CustomScrollView` to make it work with `SmartRefresher`.
It works well in 1.3.8.

But after updating to 1.3.9, error happens. It says:
```
type 'SliverGrid' is not a subtype of type 'RefreshIndicator'
```
from this line:
https://github.com/peng8350/flutter_pulltorefresh/blob/e103f8a2324fe10357937db1aecda3bd0ceeada6/lib/src/smart_refresher.dart#L191

My code is below:
```
SmartRefresher(
        enablePullDown: false,
        enablePullUp: true,
        header: WaterDropHeader(),
        controller: _refreshController,
        onLoading: () {
          // do loading
        },
        isNestWrapped: true,
        child: CustomScrollView(
          physics: RefreshBouncePhysics(),
          slivers: <Widget>[
            SliverGrid(
               ....
            )
          ],
        ));
```

Do you have any suggestion to make the code work with 1.3.9? Thank you in advance.

**Comment by peng8350** [comment]
This is a serious problem. I am uploading 1.4.0, please wait a lot

**Comment by nam-dh** [comment]
The problem is fixed on 1.4.0. Thank you.
